lib/Dialect/Seq/Transforms/LowerSeqToSV.cpp

```diff
@@ -203,6 +216,9 @@ void FirRegLower::lower() {
           });
         });
   }
+
+  if (module->hasAttr("firrtl.random_init_width"))
+    module->removeAttr("firrtl.random_init_width");
```

Comment:
`removeAttr` won't error if the attribute doesn't exist [[1]].

```suggestion
  module->removeAttr("firrtl.random_init_width");
```

[1]: https://mlir.llvm.org/doxygen/classmlir_1_1Operation.html#ae14f75cf6ddb0c5d7779343cd9988e01